task_manager/spec/factories/tasks.rb

```diff
@@ -8,6 +8,7 @@
     created_at { '2019-09-02 10:59:26' }
     priority { 1 }
     progress { 1 }
+    user { create(:user) }
```

Comment:
`FactoryBot`には関連モデルを生成する機能があるのでそちらを使ったほうが良さそうですー
```suggestion
    association :user, factory: :user
```
https://github.com/thoughtbot/factory_bot/blob/master/GETTING_STARTED.md#associations

Reply:
ありがとうございます！
確実にそちらの方がいいかと思います！そういったご意見本当に助かります、、、！

Reply:
修正しました！
ご確認お願いします！
https://github.com/Fablic/training/pull/1096/commits/548264a8354b3a10c7933f0e4dd80f841cb7cf5d